tests/functional/web/links.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Links page</title>
```

Comment:
I think this whole could be named in reference of the gecko bug. Ie. not as generic "links test page", but page for testing of this specific scenario. 

We can also maybe omit the link index.html, as this is just this single-purpose page...

Reply:
Yup - I  agree.